src/Symfony/Component/Validator/Resources/translations/validators.lt.xlf

```diff
@@ -302,10 +302,38 @@
                 <source>An empty file is not allowed.</source>
                 <target>Failas negali būti tuščias.</target>
             </trans-unit>
+            <trans-unit id="79">
+                <source>The host could not be resolved.</source>
+                <target>Serverio pavadinimas nerastas.</target>
```

Comment:
What about "Serveris negali būti pasiekiamas"?

Reply:
'Negali būti pasiekiamas' or 'nepasiekiamas' means 'unavailable'.

Reply:
Your suggested translation is correct from technical perspective, but it is not commonly used. Without an english text nearby it would be hard to understand what it means.

Reply:
Agree.